tools/dds/dds-adapter/lrs-device-controller.cpp

```diff
@@ -1536,3 +1558,125 @@ bool lrs_device_controller::on_dfu_apply( rsutils::json const & control, rsutils
 }
 
 
+bool lrs_device_controller::update_stream_trinsics( json * p_changes )
+{
+    // Returns true if any changes are detected
+    // If p_changes is not null, it should point to a json object which will be populated with stream-name:changes
+    // mappings.
+    if( p_changes && ! p_changes->is_object() )
```

Comment:
Is there a point to continue if `p_changes` is null?

Reply:
Yes: it still updates the internal intrinsics; just does not return json